server/docker/docker-compose.yml

```diff
@@ -1,5 +1,3 @@
-version: '3.8'
-
```

Comment:
`version` is removed, was getting warnings that is deprecated and needs to be removed, please verify

Reply:
I think this is ok.